src/rand.rs

```diff
@@ -402,7 +403,7 @@ mod fuchsia {
     }
 
     #[link(name = "zircon")]
-    extern "C" {
+    versioned_extern! {
```

Comment:
Isn't this a system API? Does this need to be versioned?

Reply:
This symbol should not be versioned